jenkins_jobs/build_old_host_os_versions.groovy

```diff
@@ -5,7 +5,7 @@ job('build_old_host_os_versions') {
 		"${GITHUB_ORGANIZATION_NAME}",
 		'GitHub organization from where the Host OS repositories will be checked out.')
     stringParam('BUILDS_REPO_URL',
-		"https://github.com/${GITHUB_ORGANIZATION_NAME}/builds.git",
+		"https://${GITHUB_DOMAIN}/${GITHUB_ORGANIZATION_NAME}/builds.git",
```

Comment:
Changing to `${GITHUB_DOMAIN}` (github.com) breaks the build because the SSH configuration for github (whose name change to `github` only) is not used. The next commit should be squashed into this one, so we don't have broken/unnecessary changes polluting the git history.

Reply:
The variable GITHUB_DOMAIN does not exist here. It would need to be created in the seed job to be accessible.
Since we are changing the HTTP URLs to SSH, this would be a useless change, that would need to be reverted in the next commit. It makes more sense to reorder the commits so that the change from HTTP to SSH comes first (still using the fixed github.com domain) and this commit that parameterizes the GitHub URL comes second.

By doing this, the **only** place where GITHUB_DOMAIN should appear is in the SSH config file.